server/tools/cli/jgroups/cache-owners.cli

```diff
@@ -0,0 +1,7 @@
+/subsystem=infinispan/cache-container=keycloak/distributed-cache=sessions: write-attribute(name=owners, value=${env.CACHE_OWNERS_COUNT:1})
```

Comment:
@slaskawi your test failed because you did not set JGROUPS_DISCOVERY_PROTOCOL as was required to even consider CACHE_OWNERS_COUNT / CACHE_OWNERS_AUTH_SESSIONS_COUNT.

As discussed I changed that just now and force pushed a singe commit to this branch again.
I also built and tested the Docker image myself, seems to work as you expect now.

PTAnotherL

Reply:
Ok, will do it now.